src/Game.js

```diff
@@ -1,6 +1,7 @@
 import React, { Component, Fragment, createRef } from 'react';
 
 import { ScoreBoard } from './ScoreBoard';
+import Controls from './Controls';
```

Comment:
In Controls.js as you exported the module as a named export, you have to consume it here as a named export too.

So this should be
```suggestion
import { Controls } from './Controls';
```

Reply:
Without this even your local build will fail. Check once.

Reply:
Got that.